src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
-body {
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+@import './stars';
+@import './colors';
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+button,
+img {
+  padding: 0;
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  margin: 0;
+  height: 100vh;
+
+  font-size: 10px;
+  line-height: 14px;
+
+}
+
+.card {
```

Comment:
pls move card block to a separate file and use sass bem nestind for it

Reply:
card price, card rev move too? all only the card block that going first in code?

Reply:
> pls move card block to a separate file and use sass bem nestind for it

card price, card rev move too? all only the card block that going first in code?